### DISCUSSION TITLE
Communication Timeout During Homing X/Y

### DISCUSSION THREAD

**Discussion by innovomfg** [xblax/flashforge_ad5m_klipper_mod#270] [Q&A]
I have tried all variants of the firmware releases and I still get this error.

11:36 PM
Communication timeout during homing x
11:36 PM
Communication timeout during homing x
11:36 PM
G28
11:36 PM
Communication timeout during homing y
11:36 PM
Communication timeout during homing y
11:36 PM
G28
11:36 PM
Must home axis first: 105.000 105.000 220.000 [0.000]
11:36 PM
Must home axis first: 105.000 105.000 220.000 [0.000]

**Comment by Juszzy** [comment]
I too have been getting that message randomly as of late.
But i did install 5v LED's/buck converter via the fan port on the MB. 
Also installed the FF camera.
I'm yet to test if this is caused due to power draw or similar from either.
